javascriptv3/example_code/personalize/src/personalize_createSolutionVersion.js

```diff
@@ -0,0 +1,42 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+createSolutionVersion.js demonstrates how to create a solution version (trained model) for 
+a custom dataset group with Amazon Personalize. See https://docs.aws.amazon.com/personalize/latest/dg/API_CreateSolutionVersion.html
```

Comment:
Update last sentence: 

For more information, see https://docs.aws.amazon.com/personalize/latest/dg/API_CreateSolutionVersion.html.